src/relay/api/schemas.py

```diff
@@ -280,3 +280,34 @@ class TransferInformationSchema(Schema):
     feePayer = FeePayerField(required=True, attribute="fee_payer")
     totalFees = BigInteger(required=True, attribute="total_fees")
     feesPaid = fields.List(BigInteger(), required=True, attribute="fees_paid")
+
+
+class TransferIdentifierSchema(Schema):
+    @validates_schema
+    def validate(self, data, partial, many):
+        transaction_hash = data["transactionHash"]
+        block_hash = data["blockHash"]
+        log_index = data["logIndex"]
+        if transaction_hash:
+            if block_hash or log_index:
```

Comment:
any reason why you nest those?
I would just write it as
```
if transaction_hash and (blockhash or log_index)
```
because I think less nesting is preferable.

Reply:
I think nesting is easier to understand, because it makes the following `elif` blocks more specific.

Reply:
well me and zen of python do not agree, but its not that bad, so leave it.

Reply:
Too late